src/js/util/svg.js

```diff
@@ -61,7 +61,9 @@ define(function (require, exports) {
             return null;
         }
         // currently only have icons for layers
-        if (filter.length < 2 || filter.join(" ").indexOf("LAYER") === -1) {
+        // check for libraries because of "layer styles"
+        if (filter.length < 2 || filter.join(" ").indexOf("LAYER") === -1 ||
+                filter.join(" ").indexOf("LIBRARY") !== -1) {
```

Comment:
Not sure that would work, since it's looking for if it does contain "LIBRARY" or does not contain "LAYER"